pom.xml

```
@@ -40,10 +40,10 @@
 
     <properties>
         <gravitee-bom.version>2.6</gravitee-bom.version>
-        <gravitee-node.version>1.25.0</gravitee-node.version>
+        <gravitee-node.version>2.0.0-bumpRxJava3-SNAPSHOT</gravitee-node.version>
         <gravitee-plugin.version>1.24.1</gravitee-plugin.version>
         <gravitee-alert-api.version>1.9.1</gravitee-alert-api.version>
-        <gravitee-cockpit-api.version>1.13.3</gravitee-cockpit-api.version>
+        <gravitee-cockpit-api.version>2.0.0-bumpRxJava-SNAPSHOT</gravitee-cockpit-api.version>
```

Could you please set it to the pre-released version ?
```suggestion
        <gravitee-cockpit-api.version>2.0.0-alpha.1</gravitee-cockpit-api.version>
```